userspace/otpilot/src/main.rs

```diff
@@ -169,10 +161,11 @@ fn run() -> TockResult<()> {
 
     //////////////////////////////////////////////////////////////////////////////
 
-    let mut flash_read: [u8; flash::MAX_BUFFER_LENGTH] = [0; flash::MAX_BUFFER_LENGTH];
-    writeln!(console, "reading flash ...")?;
-    flash::get().read(0x5000, &mut flash_read, 8)?;
-    writeln!(console, "flash_read = {:?}", &flash_read[0..8])?;
+    {
+        let mut flash_read: [u8; flash::MAX_BUFFER_LENGTH] = [0; flash::MAX_BUFFER_LENGTH];
```

Comment:
Somebody is likely to object to the "redundant" type specifier.

Not me.

:-)

Reply:
I removed this demo. It doesn't make any sense anymore and we don't want information from the flash to leak.

Reply:
:ack: